portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -45,14 +47,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int maxComments = maxString == null ? 5 : Integer.parseInt(maxString);
     int counter = 0;
 
-    List<String> commentHistory = new ArrayList<>();
+    Map<String, String> commentHistory = new LinkedHashMap<>();
```

Comment:
LinkedHashMap doesn't allow duplicate keys, are you intentionally disallow comments with the same name?

Reply:
Do you have a recommendation on how to allow duplicate keys? Would using a MultiMap be a good idea?

Reply:
I could also have an ArrayList of values for each key to allow for multiple comments?

Reply:
Maybe we can have a Comment class that holds name and comment, and then still use an ArrayList here?

Reply:
Would you still recommend making a separate class for a Comment with the way I have it implemented now? I created a separate class (that hasn't been pushed) but there doesn't seem to be much difference in code for the servlet.

![Screenshot 2020-07-06 at 12 26 04 PM - Display 2](https://user-images.githubusercontent.com/36430384/86632195-edd4f900-bf83-11ea-9a15-cf2775e59bdd.png)

Reply:
Sure what you currently have works too, sorry I missed it last time.